docs/source/notebooks/subset-user-input.ipynb

```diff
@@ -495,797 +495,21 @@
-      ],
-      "text/plain": [
-       "<xarray.Dataset>\n",
-       "Dimensions:  (lat: 510, lon: 1068, time: 24837)\n",
-       "Coordinates:\n",
-       "  * lat      (lat) float32 83.46 83.38 83.29 83.21 ... 41.29 41.21 41.12 41.04\n",
-       "  * lon      (lon) float32 -141.0 -140.9 -140.8 -140.7 ... -52.21 -52.13 -52.04\n",
-       "  * time     (time) datetime64[ns] 1950-01-01 1950-01-02 ... 2017-12-31\n",
-       "Data variables:\n",
-       "    tasmin   (time, lat, lon) float32 dask.array<chunksize=(7390, 51, 89), meta=np.ndarray>\n",
-       "    tasmax   (time, lat, lon) float32 dask.array<chunksize=(7390, 51, 89), meta=np.ndarray>\n",
-       "    pr       (time, lat, lon) float32 dask.array<chunksize=(7390, 51, 89), meta=np.ndarray>\n",
-       "Attributes: (12/15)\n",
-       "    Conventions:          CF-1.5\n",
-       "    title:                NRCAN ANUSPLIN daily gridded dataset : version 2\n",
-       "    history:              Fri Jan 25 14:11:15 2019 : Convert from original fo...\n",
-       "    institute_id:         NRCAN\n",
-       "    frequency:            day\n",
-       "    abstract:             Gridded daily observational dataset produced by Nat...\n",
-       "    ...                   ...\n",
-       "    dataset_id:           NRCAN_anusplin_daily_v2\n",
-       "    version:              2.0\n",
-       "    license_type:         permissive\n",
-       "    license:              https://open.canada.ca/en/open-government-licence-c...\n",
-       "    attribution:          The authors provide this data under the Environment...\n",
-       "    citation:             Natural Resources Canada ANUSPLIN interpolated hist..."
-      ]
-     },
-     "execution_count": 14,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "ImportError",
+     "evalue": "cannot import name 'Mapping' from 'collections' (/home/david/.conda/envs/pavics-sdi/lib/python3.10/collections/__init__.py)",
```

Comment:
@huard FYI, should have refreshed the notebook using the Jupyter env :)  I've done it myself since I want to be able to merge this PR to fix production.

Reply:
I don't understand what happened. I uploaded the nb on PAVICS, ran it, downloaded it then made the commit...

Reply:
Then this is really odd.  The committed version seems to be refreshed from your local machine and not from PAVICS.

Do you happen to have multiple checkout of pavics-sdi and you download the nb refreshed on PAVICS to one checkout but perform the commit from another checkout?

Or you have a cronjob that refreshes all the notebooks automatically and the scheduling happens to run after you downloaded the nb from PAVICS?